docs/syntax/frontmatter.md

```diff
@@ -43,43 +43,28 @@ See [](./applies.md)
-| `edot-cf`                                   | EDOT Cloud Forwarder                          |
-| `edot-sdk`                                  | Elastic Distribution of OpenTelemetry SDK     |
-| `edot-collector`                            | Elastic Distribution of OpenTelemetry Collector |
-| `elastic-agent`                             | Elastic Agent                                 |
-| `elastic-serverless-forwarder`              | Elastic Serverless Forwarder                  |
-| `elastic-stack`                             | Elastic Stack                                 |
-| `elasticsearch`                             | Elasticsearch                                 |
-| `elasticsearch-client`                      | Elasticsearch Client                          |
-| `filebeat`                                  | Filebeat                                      |
-| `fleet`                                     | Fleet                                         |
-| `heartbeat`                                 | Heartbeat                                     |
-| `integrations`                              | Integrations                                  |
-| `kibana`                                    | Kibana                                        |
-| `logstash`                                  | Logstash                                      |
-| `machine-learning`                          | Machine Learning                              |
-| `metricbeat`                                | Metricbeat                                    |
-| `observability`                             | Elastic Observability                         |
-| `packetbeat`                                | Packetbeat                                    |
-| `painless`                                  | Painless                                      |
-| `search-ui`                                 | Search UI                                     |
-| `security`                                  | Elastic Security                              |
-| `winlogbeat`                                | Winlogbeat                                    |
+
+:::{include} /_snippets/products-list.md
+:::
+
+`products` can also be defined in the [`docset.yml` file](/configure/content-set/navigation.md#products).
+If you define `products` in a page's Markdown file and the `docset.yml` file also includes `products`, docs-builder will combine the two lists.
+You can _not_ override doc set level `products` at the page level.
+
+:::{tip}
+`products` is distinct from `applies_to`, which is used to indicate feature availability and applicability.
+
+% Is this true?
+For example, while a page might contain content that is _applicable_ to Elastic Cloud Serverless projects,
+it might not be _about_ Elastic Cloud Serverless. In that case, you would include the `serverless` key in `applies_to`,
+but _not_ include the `cloud-serverless` ID in `products`:
```

Comment:
Could we perhaps replace this with a small table of When to use vs When not to use?

Reply:
This makes sense.

But the original reason was that `products` are only used for the search.elastic.co filters. At the time of implementation, we did not know how `applies_to` and `products` relate to.

Not sure if we even know this today. (this statement is actually a good explanation) But yes, there were ongoing discussions that they are similar. same same but different.

Reply:
@colleenmcginnis Examples, I would say. Just a suggestion though, this is not blocking the PR.

Reply:
This is a tricky example. I would add the corresponding product(s) in all cases.

Meaning that "A page with content that applies to Elastic Cloud Serverless should also be identified with the "cloud-serverless" product ID", to be searchable as such using the elastic.co search. I think this is true in most cases. If the content applies to "Stack", then you'd want product ids to identify which stack components are relevant here, etc.

Reply:
> Meaning that "A page with content that applies to Elastic Cloud Serverless should also be identified with the "cloud-serverless" product ID", to be searchable as such using the elastic.co search. 

In that case wouldn't we want that to be derived automatically from the `applies_to` at some point, rather than requiring the duplication? Then the `products` would only need to include the level of product detail (components of the stack, etc) that aren't covered in the other metadata

Reply:
But overall this seems logical, if we mark content as applicable to a context, why wouldn't we allow users to find that content when selecting that context through facets?

Reply:
I just removed this for now. We'll have to revisit it as a part of the cumulative docs initiative.